### DISCUSSION TITLE
Can `RRethy/vim-illuminate` be moved to an extra?

### DISCUSSION THREAD

**Discussion by surmish** [LazyVim/LazyVim#3187] [General]
I don't see it affecting other plugins, and looks like a nice to have plugin. Good candidate for `LazyVim extra`, no?

**Comment by folke** [comment]
Yeah, I probably will add simple lsp-only references (Neovim builtin) with working `]]` and `[[` keymaps and move it to an extra.

**Reply by folke** [reply]
Done!
